generator/README.md

```diff
@@ -0,0 +1,47 @@
+# HelloWorld
+
+> This is a React Native application developed in Reason.
+
+## Installation
+
+To get started, add the following scripts to your `package.json`:
+
+```json
+"scripts": {
+  "start": "node node_modules/react-native/local-cli/cli.js start",
+  "build": "bsb -make-world -clean-world",
+  "watch": "bsb -make-world -clean-world -w"
+}
+```
+
+Next, run `watch` and `start` scripts from the command line so that both processes are running at the same time. This can be done with `yarn` (e.g. `yarn watch`) or `npm` (e.g. `npm run watch`).
+
+The first script compiles you Reason code in a watch mode, meaning that every change is instantly recompiled.
+
+Second one is a React Native packager that serves the compiled Reason code to the React Native client.
+
+## Hiding `.bs.js` files
+
+Bucklescript compiles your Reason files to Javascript `in source`, meaning that `App.re` will result in `App.bs.js` file generated next to it.
```

Comment:
How about adding information that user can change bsconfig.json to generate .js files in lib directory?

Reply:
Oh, then... how about we just get rid of that setting by default and remove this notice?

Reply:
I am ok with both options :D